statsmodels/iolib/summary2.py

```diff
@@ -223,7 +223,7 @@ def as_latex(self):
 
         if self._merge_latex:
             # create single tabular object for summary_col
-            tab = re.sub(to_replace,'\midrule\n\midrule\n', tab)
+            tab = re.sub(to_replace,'\\\midrule\n\\\midrule\n', tab)
```

Comment:
Shouldn't it be `'\\\\midrule\n\\\\midrule\n'` or `r'\midrule\n\midrule\n'`?

Reply:
You would think so but Python 3.7 wouldn't allow `\\midrule` which should be the correct statement.

Reply:
Sorry, I meant `r'\\midrule\n\\midrule\n'`. You need to use only 2 backslashes in a raw string literal.

The `\n` escape sequences is interpreted in the replacement string, so both `'\n'` and `r'\n'` mean the same.